src/twisted/web/test/test_http.py

```diff
@@ -1871,6 +1871,43 @@ def process(self):
         self.assertEqual(processed[0].args, {b"text": [b"abasdfg"]})
 
 
+    def test_multipartFileData(self):
+        """
+        If the request has a Content-Type of C{multipart/form-data}, and the
+        form data is parseable, the form arguments will be added to the
+        request's args.
```

Comment:
Something about how `input`s of type `file` in particular are added to the request's args would be good.